gcpdiag/lint/gcf/warn_2021_001_cloudfunctions_deprecated_runtimes.py

```diff
@@ -12,8 +12,9 @@ def run_rule(context: models.Context, report: lint.LintReportRuleInterface):
   cloudfunctions = gcf.get_cloudfunctions(context)
   if not cloudfunctions:
     report.add_skipped(None, 'no functions found')
-  for _, cloudfunction in sorted(cloudfunctions.items()):
-    if cloudfunction.runtime in ['go111', 'nodejs8']:
+  for cloudfunction in sorted(cloudfunctions.values(),
+                              key=lambda cloudfunction: cloudfunction.name):
+    if cloudfunction.runtime in ['go111', 'nodejs8', 'nodejs6']:
```

Comment:
It might be a good idea to put this list as a module constant, to add visibility into it, something like:

```
DEPRECATED_RUNTIMES=['go111', 'nodejs6', 'nodejs8']
```

Also, please sort the list.